src/ArcCommander/APIs/GitAPI.fs

```diff
@@ -25,6 +17,40 @@ module GitAPI =
 
         Fake.IO.Path.getDirectory(gitDir)
 
+    /// Authenticates to a token service and stores the token using git credential manager.
+    let authenticate (arcConfiguration : ArcConfiguration) =
+
+        let log = Logging.createLogger "GitAuthenticateLog"
+
+        log.Info("Start Arc Authenticate")
+        
+        match Authentication.tryLogin log arcConfiguration with 
+        | Ok token -> 
+            log.Info($"INFO: Successfully retrieved access token from token service")
```

Comment:
You don't need interpolated strings when nothing is interpolated. 😄 
Also, no "INFO: " prefix.
-> 
```f#
log.Info("Successfully retrieved access token from token service")
```